### PR TITLE
feat: add pages of token transfer and receive

### PR DESCRIPTION
**Pull request by duanyytop**

### CODE REVIEW

File: packages/neuron-ui/package.json [hunk 1]
```diff
@@ -45,7 +45,9 @@
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.5",
     "styled-components": "^4.1.3",
-    "typescript": "3.2.2"
+    "typescript": "3.2.2",
+    "qrcode.react": "0.9.3",
+    "@types/qrcode.react": "0.8.2"
```
Comment (Keith-CY) [inline packages/neuron-ui/package.json]: `@types/xx` should be listed in `devDependencies`

File: packages/neuron-ui/src/contexts/transfer.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { createContext } from 'react'
```
Comment (Keith-CY) [inline packages/neuron-ui/src/contexts/transfer.ts]: If this really necessary to be a context?

Comment (ashchan) [reply packages/neuron-ui/src/contexts/transfer.ts]: No. Context should be at minimum.

File: packages/neuron-ui/package.json (hunk 1 above)
Comment (duanyytop) [reply packages/neuron-ui/package.json]: ok

File: packages/neuron-ui/src/components/Receive/index.tsx [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+  width: 300px;
+  margin: 20px 0 30px 150px;
+`
+
+const Address = styled.div`
+  color: black;
+`
+
+const Receive = () => {
+  const address: string = '0xcf078d66b3614C4c32B018ceF9100A39FaE7DC0D'
```
Comment (ashchan) [inline packages/neuron-ui/src/components/Receive/index.tsx]: For now please use a type hash (`H256`) for dummy address.

File: packages/neuron-ui/src/contexts/transfer.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+interface ITransfer {
```
Comment (ashchan) [inline packages/neuron-ui/src/contexts/transfer.ts]: Do not prefix `I` for interfaces.

File: packages/neuron-ui/src/contexts/transfer.ts (hunk 2 above)
Comment (ashchan) [inline packages/neuron-ui/src/contexts/transfer.ts]: We don't need a Transfer context.

We don't need context for every feature to add. :P

Comment (duanyytop) [reply packages/neuron-ui/src/contexts/transfer.ts]: If I want transfer data to be remembered, how to do ?

File: packages/neuron-ui/src/components/Receive/index.tsx (hunk 3 above)
Comment (duanyytop) [reply packages/neuron-ui/src/components/Receive/index.tsx]: ok

File: packages/neuron-ui/src/contexts/transfer.ts (hunk 4 above)
Comment (duanyytop) [reply packages/neuron-ui/src/contexts/transfer.ts]: ok, I referenced settings.ts and I will rename it.

File: packages/neuron-ui/src/contexts/transfer.ts (hunk 2 above)
Comment (duanyytop) [reply packages/neuron-ui/src/contexts/transfer.ts]: ok

Comment (ashchan) [reply packages/neuron-ui/src/contexts/transfer.ts]: The component should use local state/property to handle that. When any writings happen, they will pass through IPC channel to the electron app, and our global wallet gets updated automatically.

We'll implement viewing the wallet context as heart of the UI layer. At the moment keep in mind we just want to mock the UI flow.

Comment (duanyytop) [reply packages/neuron-ui/src/contexts/transfer.ts]: ok.I get it

**Pull request closed without merging**